syntax = "proto3";

package PB;
option java_package = "com.flipperdevices.nfc.protobuf";

message NfcaReadRequest {
  bool ok = 1;
}

message NfcaReadResponse {
  bool ok = 1;
  bytes uid = 2;
  bytes sak = 3;
  bytes atqa = 4;
  uint32 uid_len = 5;
}